iothub/service/src/Exceptions/ExceptionHandlingHelper.cs

```diff
@@ -18,102 +17,30 @@ internal class ExceptionHandlingHelper
         private const string MessageFieldErrorCode = "errorCode";
         private const string HttpErrorCodeName = "iothub-errorcode";
 
-        private static readonly IReadOnlyDictionary<HttpStatusCode, Func<HttpResponseMessage, Task<Exception>>> s_mappings =
```

Comment:
We don't have to map the returned 3-digit http status code in response to IotHubStatusCode here, so delete